packages/paste-theme/src/types/GenericThemeShape.ts

```diff
@@ -77,7 +77,7 @@ export interface GenericThemeShape {
       [key in SizingKeys]: any;
     }
   >;
-  iconSizes: {
+  iconSizes: Partial<{
```

Comment:
I really hope I didn't do this for an undocumented reason that I can no longer remember. 😟

Reply:
Me too, but this change shouldn't be harmful anyways I don't think? I hope at least